Formula/Confd.rb

```diff
@@ -0,0 +1,45 @@
+class Confd < Formula
+    desc "Manage local application configuration files using templates and data from etcd or consul"
+    homepage "https://github.com/kelseyhightower/confd"
+    url "https://github.com/kelseyhightower/confd/archive/v0.16.0.tar.gz"
+    sha256 "4a6c4d87fab77aa9827370541024a365aa6b4c8c25a3a9cab52f95ba6b9a97ea"
+    head "https://github.com/kelseyhightower/confd.git"
+  
+    depends_on "go" => :build
+  
+    def install
+      ENV["GOPATH"] = buildpath
```

Comment:
https://github.com/Homebrew/homebrew-core/issues/47627

Please try to use modules instead of GOPATH

Reply:
I am sorry, this project does not use the go modules to manage dependencies